resources/assets/sass/extern/_footer.scss

```diff
@@ -123,16 +101,8 @@
     margin-top: -2.5em;
     padding-top: 2.5em;
     .inner {
-      padding: 2em 2em 0.1em 2em;
-    }
-
-    form {
-      margin: 0 0 3em 0;
-    }
-
-    .contact {
-      margin: 0 0 3em 0;
-    }
+			padding: 2em 2em 0.1em 2em;
```

Comment:
Mijn `git diff` op deze file toonde deze aanpassing met grotere indents, terwijl dit eigenlijk niet zou moeten, want ik volg gewoon de `.editorconfig` file. Maar nu lijkt ie hier weer goed toch?

Reply:
Het kan zijn dat een aantal van deze bestanden nog niet helemaal goed kloppen qua indentie

Reply:
Hmm apart. Als ik naar Files Changed op GitHub kijk, dan lijkt het goed te staan, maar ik reageer hier net, en in het stukje laat ie dan zien dat het niet klopt.

Hoe kan ik die indentatie eigenlijk fixen? Ik dacht dat ie `.editorconfig` gewoon hoort te pakken als ik die plugin heb geinstalleerd, wat ik heb.

Reply:
De rest van die file zijn spaces, maar tabs is wat editorconfig zegt